www/js/control/general-settings.js

```diff
@@ -429,6 +432,19 @@ angular.module('emission.main.control',['emission.services',
             }
         }
     }
+    $scope.changeCarbonDataset = function() {
+        $ionicActionSheet.show({
+          buttons: CarbonDatasetHelper.getCarbonDatasetOptions(),
+          titleText: "Select a dataset for carbon footprint calculations",
+          cancelText: "Cancel",
```

Comment:
`titleText` and `cancelText` should be translated.

Reply:
So you are going to put these into the i18n file, right?

Reply:
Yes :)